.github/workflows/odh-notebooks-sync.yml

```diff
@@ -95,24 +112,36 @@ jobs:
               cd "$dir"
               minimum_supported_python_version_major=$(echo "${MINIMUM_SUPPORTED_PYTHON_VERSION}" | awk -F '.' '{print $1}') #integer of MINIMUM_SUPPORTED_PYTHON_VERSION env variable
               minimum_supported_python_version_minor=$(echo "${MINIMUM_SUPPORTED_PYTHON_VERSION}" | awk -F '.' '{print $2}') #decimal of MINIMUM_SUPPORTED_PYTHON_VERSION env variable
-              pipfile_python_version=$(grep -E '^python_version' ./Pipfile | cut -d '"' -f 2) # extracted from pipfile
+              if ! [ -f "Pipfile" ]; then
+                if [ -f "Pipfile.cpu" ]; then
+                  pipfile_python_version=$(grep -E '^python_version' ./Pipfile.cpu | cut -d '"' -f 2) # extracted from pipfile.cpu
```

Comment:
Do you need to extract version also from Pipfile.gpu?

Reply:
Actually it seems the[ script used](https://github.com/opendatahub-io/notebooks/blob/5573f26767fb41ea347a6339d01e113ebef35094/scripts/update_library_version.sh#L94) to update Pipfile.cpu and Pipfile.gpu are identical wrt python version and are being updated on same time.. so selecting pipfile version from one of them seems ok to me

Reply:
That is true - they should always match - basically the Python version should match the name of the directory.
We could probably put some comment about this here why we ignore the Pipfile.gpu value in this case, but otherwise I agree with this approach.